### ISSUE TITLE
type of 'bypass_session_save_file_types' unclear

### ISSUE THREAD

**Issue by technicalpickles** [rmagatti/auto-session#190]
The README.md says boolean: https://github.com/rmagatti/auto-session/blob/da03666a61af32eb0f57b37f18c020a3a33a44d4/README.md?plain=1#L137

doc/autosession.txt says string: https://github.com/rmagatti/auto-session/blob/da03666a61af32eb0f57b37f18c020a3a33a44d4/doc/auto-session.txt#L22

But the code is a table?

https://github.com/rmagatti/auto-session/blob/da03666a61af32eb0f57b37f18c020a3a33a44d4/lua/auto-session.lua#L197

I can make a PR to update, but figured I'd ask to confirm.

**Comment by rmagatti** [comment]
You're right 🤦 probably a result of many updates over time and a bit of a lack of attention on my part, PR with fix incoming 👍
